CuratedContent/TuringWay.md

```diff
@@ -0,0 +1,33 @@
+# The Turning Way: Reproducible and Ethical Science. 
+<!--deck text start-->
+An online handbook to reproducible, ethical, and collaborative computational and data science.
+<!--deck text end-->
+
+#### Contributed by [Jed Brown](https://github.com/jedbrown), [Mark C. Miller](https://github.com/markcmiller86 "Mark C. Miller GitHub Profile")
+#### Publication date: June 22, 2023
+
+Resource information | Details
+:--- | :--- 
+Article title  | [The Turing Way](https://the-turing-way.netlify.app/index.html), [DOI (most up to date)](https://doi.org/10.5281/zenodo.3233853)
+Authors | The Turing Way Community, (2022, Dec 21). The Turing Way: A handbook for reproducible, ethical and collaborative research. Zenodo. 
+Length | Multi-Chapter, online handbook
+Focus | Reproducible, ethical and collaborative data science
+
+This is a wonderful handbook to reproducible, ethical, and collaborative computational and data science.
+It is an online collaborative project that brings together the expertise and experiences of various contributors in the field of data science and research.
+It contains impactful illustrations and is written in a consistent, approachable style.
+Although it does not have an HPC focus and some of the details are specific to biomedical applications, the chapters on Reproducible Research, Project Design, Communication, Collaboration, and Ethical Research are largely applicable to HPC/CSE.
+
+*The Turing Way* project is open source, open collaboration, and community-driven.
+It involves and supports a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
+It's goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
```

Comment:
I was thinking of having this section quoted and then it can still use "our" like in the original. You can choose which version to accept.
```suggestion
*The Turing Way* project is open source, open collaboration, and community-driven.
It involves and supports a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
Its goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
```

Reply:
```suggestion
> *The Turing Way* project is open source, open collaboration, and community-driven.
It involves and supports a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
Our goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
```